### DISCUSSION TITLE
Email OTP

### DISCUSSION THREAD

**Discussion by TC3000** [supabase/supabase#23137] [Feature Requests]
Using phone authentication, we can set a phone number and an OTP for testing. Can we please have the same feature but for email? For example `test@email.com=123456`.

test@email.com is the email and 123456 is the OTP that will always be associated with the email.

![supabase](https://github.com/supabase/supabase/assets/89990405/67c758b5-c4ea-4c51-be95-66ad634fb0c0)

**Comment by podbasico** [comment]
I second this one here... I'm in the process of App Review for Apple and this would be very handy.

**Comment by christofferbergj** [comment]
I also second this. 

It's difficult for us to do Playwright testing without this in place. 🎭

**Comment by wouterpeere** [comment]
I also do second this. Would be useful for my CI/CD
